tmux/tmux
# Add access control list

To test:

- Build and run a shared tmux session: `make && ./tmux -S /tmp/tmux.socket -vv -f/dev/null -Ltest`
- Verify another user can't join: `./tmux -S /tmp/tmux.socket attach`
- Host allows guest to join: `allow-whitelist`
- Host allows guest to write: `allow-write`
- Host remove guest write: `deny-write`
- Host removes guest: `deny-whitelist`

This PR adds an access control list to reduce the possibility of a random individual joining a shared tmux session. 

It also adds commands for adding/removing people from the access control list as well as limiting their ability to write to the command line when joining. The owner of the session is ability to give/revoke write permissions through commands.
